march_gait_selection/src/march_gait_selection/state_machine/setpoints_gait.py

```diff
@@ -17,3 +23,39 @@ def starting_position(self):
     def final_position(self):
         subgait = self.subgaits[self.graph.end_subgaits()[0]]
         return dict([(joint.name, joint.setpoints[-1].position) for joint in subgait.joints])
+
+    def start(self):
+        self._current_subgait = None
+        self._should_stop = False
+        self._subgait_duration = 0.0
+
+    def update(self, elapsed_time):
+        if self._current_subgait is None:
+            self._current_subgait = self.graph.start_subgaits()[0]
+            self._subgait_duration = 0.0
+            return self.subgaits[self._current_subgait].to_joint_trajectory_msg(), False
+
+        self._subgait_duration += elapsed_time
+        if self._subgait_duration >= self.subgaits[self._current_subgait].duration:
+            if self._should_stop:
+                next_subgait = self.graph[(self._current_subgait, self.graph.STOP)]
+                self._should_stop = False
+            else:
+                next_subgait = self.graph[(self._current_subgait, self.graph.TO)]
```

Comment:
```suggestion
                next_subgait = self.graph[(self._current_subgait, self.graph.TO)]
```
Why is this called "TO", and not something like "NEXT"?

Reply:
That is the name I gave to the default transition of subgaits in another PR